README.md

```diff
@@ -3,111 +3,111 @@
+- Jakarta EE APIs in Java files:  [Eclipse LSP4Jakarta](https://github.com/eclipse/lsp4jakarta#eclipse-lsp4jakarta), the Language Server for Jakarta EE.
 
-## Contributing
+Liberty Tools for Visual Studio Code depends on the [Tools for MicroProfile](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-microprofile) Visual Studio Code extension. This extension provides editing assistance for MicroProfile APIs in Java and microprofile-config.properties files.
+
+## Quick Start
+
+- Install the extension.
+- Liberty supported projects will appear in the Liberty Dashboard on the side bar.
+- Right-click a project in the Liberty Dashboard to view the available commands.
+
+For minimum requirements information and detailed instructions on how to use the Liberty actions, check out the [Liberty Tools for Visual Studio Code user guide](docs/user-guide.md) page.
 
-Contributions to the Liberty Tools extension are welcome!
+## Commands
 
-Our [CONTRIBUTING](CONTRIBUTING.md) document contains details for submitting pull requests.
+The following commands are available when you select a project in the Liberty Dashboard.
 
-To build the extension locally:
+| Command | Description |
+| --- | --- |
+| Start  | Starts dev mode. |
+| Start…​ | Starts dev mode with custom parameters. Supported parameters can be found in the documentation for the [dev goal of the Liberty Maven Plugin](https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#dev) and the [libertyDev task of the Liberty Gradle Plugin](https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyDev.md#command-line-parameters). |
+| Start in a container | Starts dev mode with Liberty running in a container. The `liberty-maven-plugin` must be version `3.3-M1` or higher. The `liberty-gradle-plugin` must be version `3.1-M1` or higher. |
+| Stop | Stops dev mode. Liberty must be running in dev mode to use this command. |
+| Run tests | Runs the unit tests and integration tests that are configured for your project. Liberty must be running in dev mode to use this command. |
+| View integration test report (Maven) | Views the integration test report file it exists at `/target/site/failsafe-report.html`. |
+| View unit test report (Maven) | Views the unit test report file if it exists at `/target/site/surefire-report.html`. |
+| View test report (Gradle) | Opens the test report file, if it exists at the `build/reports/tests/test/index.html` default location. This action command is available only to Gradle projects. Gradle projects have only a single action command for test result reporting. |
 
-1. `git clone https://github.com/OpenLiberty/liberty-tools-vscode`
-2. `cd liberty-tools-vscode`
-3. Run `npm install`
-4. Run `npm run build`
-5. Run `npm run compile`
-6. Run the extension in Debug and Run mode by selecting `Run Extension` or `F5`
+## Configurable User Settings
```

Comment:
```suggestion
## Configurable user settings
```

Reply:
Where are these settings specified? Recommend adding a sentence before the table to say where/how they can be set